src/components/Header/index.js

```diff
@@ -0,0 +1,24 @@
+import Logo from '../../assets/logo.png'
+import ShoppingCart from '../../assets/shopping-cart.png'
+import styles from './styles.module.css'
+
+export default function Header () {
+    return (
+        <header>
+            <div className={styles.logo}>
+                <img src={Logo} />
+                <h3>Cornerstore</h3>
+            </div>
+            
+            
+            
+            <input type="text" className={styles.search} placeholder="Search a product"></input>
+            
+
+            <div className={styles.logo}>
+                <img src={ShoppingCart} />
```

Comment:
Be careful on how you name your assets, at first I thought it was a React component :)